inspirehep/modules/workflows/mappings/holdingpen/hep.json

```diff
@@ -57,6 +57,43 @@
                             },
                             "type": "object"
                         },
+                        "arxiv_eprints": {
+                            "properties": {
+                                "categories": {
+                                    "type": "string"
+                                },
+                                "value": {
+                                    "fields": {
+                                        "raw": {
+                                            "index": "not_analyzed",
+                                            "type": "string"
+                                        }
+                                    },
+                                    "type": "string"
+                                }
+                            },
+                            "type": "object"
+                        },
+                        "dois": {
+                            "properties": {
+                                "material": {
+                                    "type": "string"
+                                },
+                                "source": {
+                                    "type": "string"
+                                },
+                                "value": {
+                                    "fields": {
+                                        "raw": {
+                                            "index": "not_analyzed",
```

Comment:
DOIs are case insensitive, so search in DOIs should be case-insensitive.

Reply:
Right, in fact I made this mistake here too: https://github.com/inspirehep/inspire-next/blob/f64f0eee6a5a2c0eb996352f1f10371eb64cf530/inspirehep/modules/records/mappings/records/hep.json#L414

Reply:
GitHub tried to be helpful here and attached this comment to the wrong line after I force pushed... I fixed this comment, and I also added a commit to amend the records mappings.